### DISCUSSION TITLE
Multiple Arguments with complex object

### DISCUSSION THREAD

**Discussion by KrunalC28** [megahertz/electron-log#252] [Q&A]
Hi,

Is there any way that electron-log can convert passed array of texts into string and display it ?

ex. log.info('API Response is : ', apiResponse). Where apiResponse can be complex object or an error.

As of now it converts into array of passed arguments.

like
[2021-03-25 11:59:27.088] [error] [ [ 'API Response is : ', stringify of apiResponse  ] ]

**Comment by megahertz** [comment]
Hi. What is your configuration? It doesn't seem like default behavior.

**Reply by KrunalC28** [reply]
Hi,

It's angular 10 + Electron App.

I am trying to print logs of Angular Component using electron-log. So for that I have used invoke and handler mechanism through preload.js

**Reply by megahertz** [reply]
Could you send the code?

**Reply by KrunalC28** [reply]
Sure. Will create small app and share with you

**Reply by KrunalC28** [reply]
Hi,

It got resolved. We exported log from preload.js and directly used it in component rather then using invoke and handler

Thanks,
Krunal
